### DISCUSSION TITLE
How to solve H#3.5 ?

### DISCUSSION THREAD

**Discussion by Henri-FR** [thomas-maeder/popeye#506] [Ideas]
Hello,

H#3.5 is a 7 moves solution as 1... Bb6 2.Sf6 Bd4 3.Sxd3 Bc3 4.Sxe5 Bxe5# 

I try with H#3,5 doesn't work. 
If white start option in helpmate it give a solution in 6 moves (for H#3) or 8 moves (with H#4) 

The Popeye instructions doesn't explain the way to go.

Can you help?

Sincerly

Henri

**Comment by Henri-FR** [comment]
Here is the complete example of a problem in H#3.5 published in The Problemist supplement of july 2024 :

![Exemple of elpmate in 3 and a half move](https://github.com/user-attachments/assets/f93296fc-b7a6-4dc5-9c76-c7c81d1178b6)

**Comment by dardino** [comment]
Hi, you can see yout problem as a 4 moves with `WhiteToPlay` then you can add this in the options:

```text
BeginProblem
Pieces
White Bc7 Kb5 Pe5 Pe4 Pd3 Pc2 Ra3
Black Be8 Bf2 Kg3 Sh3 Sh2 Pe7 Pe6 Pb4 Pb7 Pg4 Ph4 Pg2 Qd8 Rb1 Rc6
Stipulation H#4
Option WhiteToPlay Try Set Variation
EndProblem
```

If you want, you can use this sites to compose popeyes commands:

- [ScacchiPainter](https://orange-sea-080bc3503.azurestaticapps.net/edit/1)
- [FEN Tool](https://mutsuntsai.github.io/fen-tool/)

**Comment by regis-henri** [comment]
Thanks perfect !  Thanks for your information about ScachPainter and FEN Tool to compose popeyes commands. I allready integrated them in  Sinfonia 2024 which is freeware. Actualy it is a beta version for testing. It is the follower of SuperBase because sytem changed (Now Windows 7,8,10,11, and Visual Studio coding.
